src/register/tests/RegistrationPage.test.jsx

```diff
@@ -384,7 +393,31 @@ describe('RegistrationPage', () => {
 
       expect(registrationPage.find('input#username').prop('value')).toEqual('test-user');
     });
+    it('should remove extra character if username is more than 30 character long', () => {
+      const registrationPage = mount(reduxWrapper(<IntlRegistrationPage {...props} />));
+      registrationPage.find('input#username').simulate('change', { target: { value: 'why_this_is_not_valid_username_', name: 'username' } });
 
+      expect(registrationPage.find('input#username').prop('value')).toEqual('');
+    });
+
+    it('should give error with suggestion for common top level domain mistakes', () => {
+      const registrationPage = mount(reduxWrapper(<IntlRegistrationPage {...props} />));
+      registrationPage.find('input#email').simulate('change', { target: { value: 'ahtesham@hotmail', name: 'email' } });
+      registrationPage.find('input#email').simulate('blur');
+
+      expect(registrationPage.find('.alert-text').text()).toEqual('Did you mean ahtesham@hotmail.com?');
```

Comment:
nit: move this received message into a variable

Reply:
You mean like this ?
const receievedMessage = 'Did you mean ahtesham@hotmail.com?';
expect(registrationPage.find('.alert-text').text()).toEqual(receievedMessage);